tests/RuleSetTest.php

```diff
@@ -607,4 +654,54 @@ private static function getRuleSetDefinitionsToTestWith()
 
         return $testSet;
     }
+
+    /**
+     * @param string $setName
+     * @param string $ruleName
+     * @param string $targetVersion
+     */
+    private static function assertVersionIsLargestAvailable($setName, $ruleName, $targetVersion)
+    {
+        $maximumPHPUnitVersion = preg_replace('/^@PHPUnit(\d)(\d)Migration:risky$/', '$1.$2', $setName);
+
+        $factory = new FixerFactory();
+        $factory->registerBuiltInFixers();
+        $factory->useRuleSet(new RuleSet([$ruleName => true]));
+        $fixer = current($factory->getFixers());
+
+        foreach ($fixer->getConfigurationDefinition()->getOptions() as $option) {
+            if ('target' === $option->getName()) {
+                $availableVersions = $option->getAllowedValues();
+                $availableVersions = array_diff($availableVersions, ['newest']);
+
+                break;
+            }
+        }
+
+        $availableVersions = array_filter(
```

Comment:
`$availableVersions` will be empty if no `target` option was defined for fixer

Reply:
Luckily all the rules for PHPUnit migrations have `target` option, handling a fixer without it looks like YAGNI to me.

Reply:
hoping to find an item in collection and not handling scenario when it's not there looks like asking for troubles in the future.
1. nobody will remember that when one will add new phpunit rule that will not have target option, he has to go to ruleset test and adjust sth not related
2. when option will not be found, we will not return string but null in current implementation, leading to type violation of method prototype

Reply:
Allright, I've added the checking.